### PR TITLE
validation

### PR DESCRIPTION
**Pull request by thuanguy**
I created the validation folder including:
- .py file for the codes
- readme.md for the process notes and findings
- .png files of histograms of distance to closest official points from each OSM for each city looked at.

### CODE REVIEW

**Review by gboeing — APPROVED** [review]

**Comment by gboeing** [comment]
@thuanguy can you resolve the branch conflicts? Then I'll merge it.

File: validation/POIs_validation.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+POIs_name = "fresh_food_market"
+
+gdf = gpd.read_file(OSM_filename, layer='destinations')
+OSM_points = gdf[gdf['dest_name']==POIs_name]
+
+OPoints = gpd.GeoDataFrame.from_file(official_filename)
+print("number of OSM points: " ,len(OSM_points))
+print("number of Official points: ", len(OPoints))
+
+#Calculate number of OSM_points intersect with OPoints
```
Comment (gboeing) [inline validation/POIs_validation.py]: Make sure you check that the two GeoDataFrame crs attributes are equal to each other before proceeding with spatial analysis. If they differ, project one to the other's CRS.

**Review by gboeing — CHANGES_REQUESTED** [review]

File: validation/POIs_validation.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+from geopy.distance import great_circle
+
+official_filename = "Restaurant_and_Food_Related.shp"
+OSM_filename = "belfast_gb_2019_1600m_buffer.gpkg"
+POIs_name = "fresh_food_market"
+
+gdf = gpd.read_file(OSM_filename, layer='destinations')
+OSM_points = gdf[gdf['dest_name']==POIs_name]
+
+OPoints = gpd.GeoDataFrame.from_file(official_filename)
```
Comment (gboeing) [inline validation/POIs_validation.py]: Can you give your variables more intuitive names? like gdf_osm and gdf_official? Easier to follow along...

**Comment by gboeing** [comment]
@thuanguy checking back on the changes requested last week...

**Comment by thuanguy** [comment]
I did resolve the conflicts. Have you seen it resolved?

**Comment by gboeing** [comment]
Not the conflicts, the changes requested in the GitHub review... if you've resolved those you need to mark them resolved above or just document here how you resolved them so I can mark them.

File: validation/POIs_validation.py (hunk 1 above)
Comment (thuanguy) [reply validation/POIs_validation.py]: Moved the projection up before intersections

File: validation/POIs_validation.py (hunk 2 above)
Comment (thuanguy) [reply validation/POIs_validation.py]: Changes made

**Comment by thuanguy** [comment]
I see. I did mark then resolved.

**Comment by gboeing** [comment]
@thuanguy did you close this pull request? Should I not merge it anymore?

**Comment by thuanguy** [comment]
Am I supposed not to close?

**Comment by thuanguy** [comment]
Sorry, I just realized I am not supposed to close. I just made another pull
request. It should include the previous commits as well as the new working
script that I added. Please let me know if it works.

**Pull request closed without merging**